virttest/utils_conn.py

```diff
@@ -743,7 +743,10 @@ def conn_recover(self):
                                             r"[\#\$]\s*$")
             libvirtd_service = utils_libvirtd.Libvirtd(session=session)
             if libvirt_version.version_compare(5, 6, 0, session):
+                session.cmd("systemctl stop libvirtd.socket")
+                libvirtd_service.stop()
                 session.cmd("systemctl stop libvirtd-tcp.socket")
+                session.cmd("systemctl daemon-reload")
```

Comment:
@cliping is it necessary to start libvirtd.socket after deamon-reload?

Reply:
Updated. Thanks.